ext/ITensorNetworksEinExprsExt/src/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,58 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors: Index, ITensor, Algorithm
+using ITensorNetworks:
+  ITensorNetworks,
+  ITensorNetwork,
+  vertextype,
+  vertex_data,
+  externalinds,
+  contraction_sequence
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function prepare_einexpr(tn::ITensorNetwork)
```

Comment:
Would prefer `einexpr_and_inds_to_vertex` or whatever name we decide to give the current `tensor_map` data structure to be clearer about what this is outputting, since `prepare` is pretty vague.

Reply:
What do you think of `to_einexpr`?

Reply:
For this I would still prefer a longer more explicit name saying what it is outputting, since it is being used internally so the priority would be on readability/understanding the code later on rather than having a short name.